### PR TITLE
i18n Changes (3/n)

### PR DESCRIPTION
**Pull request by marco**
## What changed (plus any additional context for devs)

This is the third batch of i18n changes. This is the last major batch, although there will definitely be a few places I missed.

## PoW (screenshots / screen recordings)

App runs as expected:

Screen Shot 2022-03-07 at 1 48 59 PM

```
yarn check-translations
Checking file "./src/languages/_english.json":
    ✅ No errors found for "./src/languages/_english.json".

Checking file "./src/languages/_french.json":
    ✅ No errors found for "./src/languages/_french.json".

Note, this script currently **does not** support interpolated translations such as `lang.t(`my.${variable}.based.${path}`).
    🔶 Manual check required for "lang.t(`time.${unit}.${length}.${plurality}`)".
```

## Dev checklist for QA: what to test

- Ensuring that app compiles and that there are not "missing translation" errors.

## Final checklist

- [x] Assigned individual reviewers?
- [x] Added labels?

### CODE REVIEW

**Comment by tchayen** [comment]
@marco you can merge latest develop to your branch to fix failing CI.

File: src/components/AddFundsInterstitial.js [hunk 1]
```diff
@@ -231,7 +231,9 @@ const AddFundsInterstitial = ({ network }) => {
       {network === networkTypes.mainnet ? (
         <Fragment>
           <Title>
-            To get started, buy some ETH{ios ? ` with Apple Pay` : ''}
+            {ios
+              ? lang.t('add_funds.to_get_started_ios')
+              : lang.t('add_funds.to_get_started_non_ios')}
```
Comment (skylarbarrera) [inline src/components/AddFundsInterstitial.js:236]: can we change this to _android instead of _non_ios

File: src/components/list/ListHeader.js [hunk 2]
```diff
@@ -124,7 +129,7 @@ export default function ListHeader({
           {children}
         </Content>
         {
-          /* 
+          /*
```
Comment (skylarbarrera) [inline src/components/list/ListHeader.js:132]: 💆

File: src/components/settings-menu/BackupSection/AlreadyBackedUpView.js [hunk 3]
```diff
@@ -238,7 +249,10 @@ export default function AlreadyBackedUpView() {
               size="large"
               weight="semibold"
             >
-              􀙶 Back up to {cloudPlatform}
+              􀙶{' '}
```
Comment (skylarbarrera) [inline src/components/settings-menu/BackupSection/AlreadyBackedUpView.js:252]: confirming this is on purpose

File: src/components/settings-menu/BackupSection/AlreadyBackedUpView.js [hunk 4]
```diff
@@ -252,7 +266,10 @@ export default function AlreadyBackedUpView() {
               size="lmedium"
               weight="semibold"
             >
-              􀍢 Manage {cloudPlatform} Backups
+              􀍢{' '}
```
Comment (skylarbarrera) [inline src/components/settings-menu/BackupSection/AlreadyBackedUpView.js:269]: same here just confirming

File: src/components/settings-menu/BackupSection/WalletSelectionView.js [hunk 5]
```diff
@@ -219,7 +223,10 @@ const WalletSelectionView = () => {
               size="lmedium"
               weight="semibold"
             >
-              􀍢 Manage {cloudPlatform} Backups
+              􀍢{' '}
```
Comment (skylarbarrera) [inline src/components/settings-menu/BackupSection/WalletSelectionView.js:226]: same

**Review by skylarbarrera — APPROVED** [review]
🙏

File: src/components/expanded-state/unique-token/UniqueTokenExpandedStateHeader.tsx [hunk 6]
```diff
@@ -342,11 +346,17 @@ const UniqueTokenExpandedStateHeader = ({
 
   const onPressAndroidAsset = useCallback(() => {
     const androidContractActions = [
-      'View On Web',
-      // @ts-expect-error network could be undefined?
-      `View on ${startCase(ethereumUtils.getBlockExplorer(asset?.network))}`,
-      ...(isPhotoDownloadAvailable ? (['Save to Photos'] as const) : []),
-      'Copy Token ID',
+      lang.t('expanded_state.unique_expanded.view_on_web'),
+      lang.t('view_on_block_explorer', {
```
Comment (jinchung) [inline src/components/expanded-state/unique-token/UniqueTokenExpandedStateHeader.tsx:350]: is this meant to be `expanded_state.unique_expanded.view_on_block_explorer`

File: src/components/expanded-state/unique-token/saveToCameraRoll.js [hunk 7]
```diff
@@ -23,9 +26,9 @@ const getPermissionAndroid = async () => {
 
 function alertError(err) {
   Alert.alert(
-    'Failed to save Image',
+    lang.t('expanded_state.unique.save.failed_to_save_image'),
```
Comment (jinchung) [inline src/components/expanded-state/unique-token/saveToCameraRoll.js:29]: This is not a change, but the casing seems weird here to me of the original text "Failed to save Image" @skylarbarrera @christianbaroni did we want to capitalize all or lowercase the rest of the words?

File: src/components/savings/SavingsListRowEmptyState.js [hunk 8]
```diff
@@ -42,7 +43,7 @@ const SavingsListRowEmptyState = ({ onPress }) => {
         size="lmedium"
         weight="bold"
       >
-        $0.00
+        {lang.toCurrency(0)}
```
Comment (jinchung) [inline src/components/savings/SavingsListRowEmptyState.js:46]: leaving a note for myself here to double check on other currency symbols being hooked up for this functionality

File: src/components/settings-menu/BackupSection/AlreadyBackedUpView.js [hunk 9]
```diff
@@ -238,7 +249,10 @@ export default function AlreadyBackedUpView() {
               size="large"
               weight="semibold"
             >
-              􀙶 Back up to {cloudPlatform}
+              􀙶{' '}
+              {lang.t('back_up.cloud.back_up_to_platform', {
```
Comment (jinchung) [inline src/components/settings-menu/BackupSection/AlreadyBackedUpView.js:253]: I don't see any `back_up.cloud.back_up_to_platform` key in _english.json

File: src/screens/SendSheet.js [hunk 10]
```diff
@@ -624,20 +625,22 @@ export default function SendSheet(props) {
...
     } else if (!isZeroAssetAmount && !isSufficientGas) {
       disabled = true;
-      label = `Insufficient ${nativeToken}`;
+      label = lang.t('button.confirm_exchange.insufficient_token', {
+        tokenName: nativeToken,
+      });
     } else if (!isValidGas) {
       disabled = true;
-      label = 'Invalid fee';
+      label = lang.t('button.confirm_exchange.invalid_fee');
```
Comment (jinchung) [inline src/screens/SendSheet.js:637]: No change required just making a note that I confirm the change that `Invalid Fee` matches the casing pattern of these alerts (previously it was lowercase `fee`)

File: src/screens/WalletConnectRedirectSheet.js [hunk 11]
```diff
@@ -27,12 +28,12 @@ const emojisMap = {
...
-  'reject': 'Connection canceled',
-  'sign': 'Message signed!',
-  'sign-canceled': 'Transaction canceled!',
-  'transaction': 'Transaction sent!',
-  'transaction-canceled': 'Transaction canceled!',
+  'connect': lang.t('walletconnect.titles.connect'),
+  'reject': lang.t('walletconnect.titles.reject'),
+  'sign': lang.t('walletconnect.titles.sign'),
+  'sign-canceled': lang.t('walletconnect.titles.sign_canceled'),
+  'transaction': lang.t('walletconnect.titles.transaction'),
```
Comment (jinchung) [inline src/screens/WalletConnectRedirectSheet.js:35]: would be nice to rename this to `walletconnect.titles.transaction_sent`

File: src/screens/ExplainSheet.js [hunk 12]
```diff
@@ -106,13 +97,13 @@ export const explainers = network => ({
...
     text: FLOOR_PRICE_EXPLAINER,
-    title: 'Collection floor price',
+    title: lang.t('explain.floor_price.title'),
   },
   gas: {
     emoji: '⛽️',
     extraHeight: 2,
     text: gasExplainer(network),
-    title: `${network} network fee`,
+    title: lang.t('explain.gas.title'),
```
Comment (jinchung) [inline src/screens/ExplainSheet.js:106]: This is missing the `networkName` argument

**Review by jinchung — CHANGES_REQUESTED** [review]
Left comments for some blockers!

File: src/components/settings-menu/BackupSection/AlreadyBackedUpView.js (hunk 3 above)
Comment (marco) [reply src/components/settings-menu/BackupSection/AlreadyBackedUpView.js:252]: Correct, there is an SF Symbols icon of a cloud at the beginning of that line (which GitHub doesn't seem to render) so the space comes after that icon.

File: src/components/expanded-state/unique-token/UniqueTokenExpandedStateHeader.tsx (hunk 6 above)
Comment (marco) [reply src/components/expanded-state/unique-token/UniqueTokenExpandedStateHeader.tsx:350]: @jinchung Yes, fixing now!

File: src/screens/WalletConnectRedirectSheet.js (hunk 11 above)
Comment (marco) [reply src/screens/WalletConnectRedirectSheet.js:35]: @jinchung Updated, thanks!

File: src/screens/ExplainSheet.js (hunk 12 above)
Comment (marco) [reply src/screens/ExplainSheet.js:106]: @jinchung Fixed now, thanks for catching that!

**Comment by marco** [comment]
@jinchung @skylarbarrera Thanks for the feedback above, I went through and made those changes. Please let me know if you have any other suggestions!

**Review by brunobar79 — APPROVED** [review]
Agreed with previews comments and all of them were addressed  👍

**Review by jinchung — APPROVED** [review]
:taco:

amazing work!!

**Review by skylarbarrera — APPROVED** [review]
:shipit:

**Review by ibrahimtaveras00 — CHANGES_REQUESTED** [review]
Only two issues here: 

- Issue 1: 
Share button has an extra }

image

- Issue 2:
In the savings section, there's an extra \ (view rainbowwallet.eth)

image

**Review by ibrahimtaveras00 — APPROVED** [review]
Fixes look good, QA Passed 👍🏽

**Pull request merged**